config/routes.rb

```diff
@@ -2755,9 +2755,9 @@
   API_VERSION_REGEX = /v[\d]+(\.[\da-zA-Z]+)*(\-[\da-zA-Z]+)?/ unless defined?(API_VERSION_REGEX)
 
   # OPTIONS requests for REST API pre-flight checks
-  match '/api/*path' => 'manage_i_q/a_p_i/base#handle_options_request', :via => [:options]
+  match '/api/*path' => 'a_p_i/base#handle_options_request', :via => [:options]
 
-  get '/api(/:version)' => 'manage_i_q/a_p_i/base#show_entrypoint', :format => 'json', :version => API_VERSION_REGEX
+  get '/api(/:version)' => 'a_p_i/base#show_entrypoint', :format => 'json', :version => API_VERSION_REGEX
```

Comment:
@abellotti incorporated in https://github.com/ManageIQ/manageiq/pull/10823/commits/ab855ce2c0a01f39f136060e6ae8fd59d40e6e8d